sdk/keyvault/keyvault-admin/CHANGELOG.md

```diff
@@ -2,6 +2,18 @@
 
 ## 4.0.0-beta.4 (Unreleased)
 
+- Breaking changes:
```

Comment:
Use the headings described here: https://azure.github.io/azure-sdk/policies_releases.html#change-logs

Even if that changes the format of this file for only this entry, that's fine. These headings will be processed ~next month by the release system.